app/controllers/articles_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ArticlesController < ApplicationController
+  before_filter :authenticate_user!
```

Comment:
this should be in applicationController, so that we dont have to add in all controllers